[#177] [Generate project with Mason] Update `Use the template` section in README to write the instruction for generating the project with Mason (#190)
nimblehq/flutter_templates: README.md

Checkout the [Wiki](https://github.com/nimblehq/flutter-templates/wiki) page to access the full documentation.

## Template Usage

Before using the template, make sure you have the following prerequisites installed on your system:

- [Mason CLI](https://pub.dev/packages/mason_cli) 0.1.0-dev.44
- Flutter 3.0.5
- Flutter version manager (recommend): [fvm](https://fvm.app/)

To set up a new project from the template, follow these steps:

- Open your terminal and run the following command:

>   - `$ mason make template`

Reviewer: I think we should mention the Mason doc as a reference for the `make` command usage

```
- Open your terminal and run the following command:

  - `$ mason make template`

> Checkout the Mason documation for `make` command options & usage in detail [here](https://github.com/felangel/mason/tree/master/packages/mason_cli#overview)
```

>   - `$ mason make template`

Author: @luongvo since we don't push `mason-lock.json` to the version control. It is valid to call `mason get`. I'll update 👍

- Navigate to the newly created project using the following command:

  - `$ cd {project_name}`

- Clean the project by running the command:

  - `$ fvm flutter clean`

- Re-fetch the project by running the command:

  - `$ fvm flutter pub get`

- Run code generator using the command:
  - `$ fvm flutter packages pub run build_runner build --delete-conflicting-outputs`

> - Once you have completed these steps, your new Flutter project is ready to develop!

Reviewer: We don't need to repeat what we already mentioned in the template project's README, we might need to refer to the output project & follow the README to make it ready.

```
- The new project is generated at `/{project_name}`.
- Once completed, please follow your new Flutter project is ready to develop!

```

Reviewer: By using this link https://github.com/nimblehq/flutter-templates/tree/feature/mason-integration/bricks/template/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D#getting-started, we might need to update it after bringing the Mason migration to `develop` branch.

Author: @luongvo noted, I will create a task to update this link once migrating to `develop` successfully

### Setup a new project

Before using the template, ensure that you have installed the following prerequisites on your system:

> - [Mason CLI](https://pub.dev/packages/mason_cli) 0.1.0-dev.44

Author: @luongvo so we need to add `Dart` or `Flutter` to the list I think

> - [Mason CLI](https://pub.dev/packages/mason_cli) 0.1.0-dev.44

Reviewer: @manh-t how about this? 🤔 

```
- [Dart SDK](https://dart.dev/get-dart) (to generate only) or [Flutter SDK] includes Dart (https://docs.flutter.dev/get-started/install) (to generate and develop)
```

Follow these steps to set up a new project from the template:

- Fetch all required bricks by running the command:

  - `$ mason get`

- Generate the new project by running the following command:

  - `$ mason make template`

  > You can find the detailed information on `make` command options and usage in the [Mason documentation](https://github.com/felangel/mason/tree/master/packages/mason_cli#overview).

> - Once the project is generated at `/{project_name}`, please refer to the [Getting Started](https://github.com/nimblehq/flutter-templates/tree/develop/bricks/template/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D#getting-started) documentation to prepare your project for development.

Reviewer: we can reduce the `project` word repeatedly here

```suggestion
- Once the project is generated at `/{project_name}`, please refer to the [Getting Started](https://github.com/nimblehq/flutter-templates/tree/develop/bricks/template/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D#getting-started) documentation to make it ready for development.
```

> That's it! You have now set up a new Flutter project using the template.

Reviewer: make it a bit cool with 🎉 ? 😄 

```suggestion
That's it! You have now set up a new Flutter project using the template 🎉 
```
